ale_linters/markdown/textlint.vim

```diff
@@ -0,0 +1,23 @@
+" Author: tokida https://rouger.info
+" Description: textlint, a proofreading tool (https://textlint.github.io/)
+
+function! ale_linters#markdown#textlint#GetCommand(buffer) abort
+    let l:cmd_path = ale#path#FindNearestFile(a:buffer, '.textlintrc')
+    if !empty(l:cmd_path)
+        return 'textlint'
+        \    . ' -c '
+        \    . l:cmd_path
+        \     . ' -f json %t'
+    else
+        return "[{\'filePath\':\'%t\',\'messages\':[]}]"
```

Comment:
You need to return a command to run here, not the output of it. You should return `''` instead, which will skip running the command. Write a return statement at the end of the function, instead of putting it in an `else` block.